build/stage-build-security-android.yml

```diff
@@ -0,0 +1,43 @@
+parameters:
+- name: applicationEnvironment
+  type: string
+  default: ''
+- name: androidKeyStoreFile
+  type: string
+  default: ''
+- name: androidVariableGroup
+  type: string
+  default: ''
+- name: artifactName
+  type: string
+  default: ''
+- name: pathToSrc
+  type: string
+  default: '$(Build.SourcesDirectory)/src'
+- name: androidKeyPropertiesFile
+  type: string
+  default: ''
+- name: projectName
+  type: string
+  default: '$(ProjectName)'
+- name: firebaseJsonFile
+  type: string
+- name: firebaseOptionsDartFile
+  type: string
+- name: googleServicesJsonFile
+  type: string
+- name: enableSecurityScan
+  type: boolean
+  default: false
+
+jobs:
+  - job: OnLinux_Android_SecurityScan
+    condition: eq(${{parameters.enableSecurityScan}}, true)
+    dependsOn: []
+    pool:
+      vmImage: $(ubuntuHostedAgentImage)
+    steps:
+      - template: templates/mobsf-android-scan.yml
+        parameters:
+          mobSfApiKey: '8181'
```

Comment:
Also, shouldn't it live outside source control? It doesn't look like a real API key so I don't know if there's a real risk, however if it is, it should be rotated and removed from source control.

Reply:
Not a real risk in this context because this is running in a contained environment (API key is for authentication purposes). The parameter could be omitted though (we're setting the same default value in the step YAML).

Reply:
set the key as a variable instead